internal/exec/util/passwd.go

```diff
@@ -171,13 +172,20 @@ func (u Util) AuthorizeSSHKeys(c types.PasswdUser) error {
 		if !strings.HasSuffix(ks, "\n") {
 			ks = ks + "\n"
 		}
-
+		var path string
 		if distro.WriteAuthorizedKeysFragment() {
-			err = writeAuthKeysFile(usr, filepath.Join(usr.HomeDir, ".ssh", "authorized_keys.d", "ignition"), []byte(ks))
+			path = filepath.Join(usr.HomeDir, ".ssh", "authorized_keys.d", "ignition")
+			err = writeAuthKeysFile(usr, path, []byte(ks))
 		} else {
-			err = writeAuthKeysFile(usr, filepath.Join(usr.HomeDir, ".ssh", "authorized_keys"), []byte(ks))
+			path = filepath.Join(usr.HomeDir, ".ssh", "authorized_keys")
+			err = writeAuthKeysFile(usr, path, []byte(ks))
+		}
+		if err == nil {
+			journal.Send(fmt.Sprintf("ssh authorized keys were provided via ignition for user:%s", c.Name), journal.PriInfo, map[string]string{
+				"IGNITION_USER_NAME": c.Name,
+				"IGNITION_DIR_PATH":  path,
```

Comment:
should we come up with a unique message ID we can specify to query for this message?